src/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs

```diff
@@ -387,6 +387,25 @@ public sealed class SocketsHttpHandler_HttpClientHandler_ConnectionPooling_Test
             }
         }
 
+        [Fact]
+        public async Task ServerSendsGarbageAfterInitialRequest_SubsequentRequestUsesDifferentConnection()
```

Comment:
This test is SocketsHttpHandler specific? What do the other handlers do?

Reply:
I don't know.  I added this here because the other tests that test similar connection pooling behavior are here.  

There's a comment in the class:

```csharp
        // TODO: Currently the subsequent tests sometimes fail/hang with WinHttpHandler / CurlHandler.
        // In theory they should pass with any handler that does appropriate connection pooling.
        // We should understand why they sometimes fail there and ideally move them to be
        // used by all handlers this test project tests.
```

There doesn't seem to be an issue on this, though.  I'll file one.

Reply:
Added #27272